obsRecorder.js

```diff
@@ -181,7 +196,14 @@ function setupScene() {
 }
 
 function getAudioDevices(type, subtype) {
-  const dummyDevice = osn.InputFactory.create(type, subtype, { device_id: 'does_not_exist' });
+  const dummyDevice = osn.InputFactory.create(
+    byOS({ [OS.Windows]: 'wasapi_input_capture', [OS.Mac]: 'coreaudio_input_capture' }),
+    uuid(),
```

Comment:
`getAudioDevices` is being used to get both input and output sources, so it is better to detect of OS-specific types in places that call this function. And here just use `type` and `subtype`. Btw why `uuid()` is used here instead of provided `subtype`?

Reply:
Good point, I reverted that change